### PR TITLE
Created Mongoose objects for database.

### PR DESCRIPTION
**Pull request by michaellibucha**

### CODE REVIEW

File: packages/model/src/author.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Model, Schema, model } from 'mongoose';
+
+export interface IAuthor {
```
Comment (obber) [inline packages/model/src/author.ts:3]: We will use the [TS style guide](https://google.github.io/styleguide/tsguide.html#identifiers) from Google:

> Do not mark interfaces specially (IMyInterface or MyFooInterface) unless it's idiomatic in its environment. When introducing an interface for a class, give it a name that expresses why the interface exists in the first place (e.g. class TodoItem and interface TodoItemStorage if the interface expresses the format used for storage/serialization in JSON).

I can't think of a great alternative though.... if we can't think of one, then I think `IAuthor` is fine.

File: packages/model/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@bookwave/model",
```
Comment (obber) [inline packages/model/package.json:2]: Didn't realize you would put this inside of its own package - maybe I misunderstood what you previously asked me.

I think this should go inside of the server, unless we knew that some other backend service was going to use it.

But I'm open to keeping it here, WDYT

**Review by obber — APPROVED** [review]

File: packages/model/src/author.ts (hunk 1 above)
Comment (michaellibucha) [reply packages/model/src/author.ts]: After thinking about this, I think this interface should really be the "Author" type that let's us pass an Author from the client to the server. I am planning on calling this Author and changing the stuff below to be more specific referring to the Mongoose paradigm. In a separate change, I plan on moving this, along with Chapter and Book, to a new package to be shared across client and server.

File: packages/model/package.json (hunk 2 above)
Comment (michaellibucha) [reply packages/model/package.json]: I agree with you on this one. I'll move everything over to the server folder for this change.

**Pull request merged**